app/store/bot.data.ts

```diff
@@ -117,7 +117,6 @@ export const createEmptyBot = (): Bot => ({
   createdAt: Date.now(),
   botHello: Locale.Store.BotHello,
   session: createEmptySession(),
-  datasource: "",
```

Comment:
can't we add the default in createEmptyBot? then `datasource` would be always defined